PilotDataPlatform/metadata
# PILOT-1481 Implement GET endpoint for getting favourites

## Summary

- Add `GET /v1/favourites` endpoint for getting favourite items/collections by user.

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1481

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

1. Manually create some favourites.
2. Get those favourites via the new endpoint and check that the responses are correct.
3. Review the [API design](https://indocconsortium.atlassian.net/wiki/spaces/PILOT/pages/3189309472/Favourites+API+design) and ensure that the implementation matches the design.
